content/30.abstracting_simulation_types.md

```diff
@@ -86,7 +86,26 @@ Scheme of the AMR structure used to estimate the gradient of a quantity in the c
 
 ### SPH Analysis
 
-Smoothed Particle Hydrodynamics (SPH) is **MJT: We need a very brief explanation of SPH here!**
+Smoothed Particle Hydrodynamics (SPH) is a commonly-used method for solving equations of hydrodynamics in astrophysics (as well as many other fields!) from a lagrangian perspective.
+This provides many advantages over grid-based discretizations, but also poses somewhat different challenges for analysis and visualization.
+While a full description of SPH is outside the scope of this paper, there are a handful of crucial and important pieces of information that we will review.
+For more information, we refer interested readers to this comprehensive review of Smoothed Particle Hydrodynamics by Volker Springel [@doi:10.1146/annurev-astro-081309-130914] or to the SPLASH method paper by Daniel Price [@doi:10.1071/AS07022].
+SPH defines field quantities *at* a set of moving points, and field values themselves can be computed by integrating over all the elements in the domain using a special-purpose kernel.
```

Comment:
not sure i understand distinction between field quantities and "field values themselves" unless you mean "field values between these discretized points." If you do, then I'd try and emphasize that what you're doing here is an exact interpolation between the discretized points using the kernel.